specification/sbi_atee.adoc

```diff
@@ -943,10 +804,9 @@ The possible error codes returned in `sbiret.error` are shown below.
 | SBI_ERR_FAILED        | The operation failed for unknown reasons.
 |===
 
-The TSM updates the `scause` field in the `tvm_vcpu_supervisor_csrs` region in the
-shared-memory for the VCPU that was previously configured by the host. The host should
+The TSM updates the `scause` field in the NACL shared memory region. The host should
 use the `scause` field to determine whether the exit was caused by an interrupt or exception,
-and then use the additional information in the shared-memory region to determine further
+and then use the additional information in the NACL shared memory region to determine further
 course of action (if sbiret.value is 0).
 
 The TSM sets the most significant bit in `scause` to indicate that the exit was caused
```

Comment:
The following section can be removed as it just reiterates what priv spec already says.

Reply:
I have seen that in multiple places. Planning to go through the whole spec and will fix this in that PR.

Reply:
+1 the comment sections are useful IMO so should keep the listing for description.